invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/controls/NewVersionButton/NewVersionButton.js

```diff
@@ -28,23 +28,29 @@ export const NewVersionButton = ({ onError, record, disabled, ...uiProps }) => {
 
   return (
     <Popup
-      content={i18next.t("You don't have permissions to create a new version.")}
+      content={i18next.t(
+        "Versioning is disabled for records that use a non-Zenodo DOI"
```

Comment:
Nice message, however we cannot put `Zenodo` name in `InvenioRDM` :(

Reply:
Good point, I removed the last commit with the rewording and left the existing more generic message.
This pull request is then only about fixing the tooltip in InvenioRDM, and I will later check if I can override the message in Zenodo.